KDICCore/KDICPlayer.swift

```diff
@@ -20,6 +20,7 @@ open class KDICPlayer {
     player.pause()
   }
   
+    /** to fix **/
```

Comment:
We've made a ticket, so you can remove this comment.

Alternatively, if you'd like to keep it, we should use proper Swift documentation syntax. You can find information on that [here](http://nshipster.com/swift-documentation/). (Hint: There is a tag called `Bug`.)